package.json

```diff
@@ -73,8 +73,9 @@
       "local-simulator": "quantum/nodes/local-simulator/local-simulator.js",
       "ibm-quantum-system": "quantum/nodes/ibm-quantum-system/ibm-quantum-system.js",
       "circuit-diagram": "quantum/nodes/circuit-diagram/circuit-diagram.js",
-      "bloch-sphere": "quantum/nodes/bloch-sphere/bloch-sphere.js",
-      "script": "quantum/nodes/script/script.js"
+      "script": "quantum/nodes/script/script.js",
+      "grovers":  "quantum/nodes/grovers/grovers.js",
```

Comment:
Can we move the Grover node to a new folder named 'quantum-algorithms' next to the 'quantum' folder. Would make more sense wince we have 2 libraries in the palette now